src/hive/components/screens/home/core.cljs

```diff
@@ -111,7 +111,7 @@
                       :showsUserLocation true :style {:flex 1}
                       :showsMyLocationButton true}]))
 
-(defn home
+(defn Home
```

Comment:
@carocad are you using Pascal case for naming react components?

Reply:
I wasnt before. Now I think it would be better :)

Reply:
and it is also the way that most javascript react native projects work. So I think it would make things easier